src/parser/avc_parser.h

```diff
@@ -125,6 +125,10 @@ class AvcVideoParser : public RocVideoParser {
     int curr_ref_pic_bottom_field_;
     int max_long_term_frame_idx_; // MaxLongTermFrameIdx
 
+    /frame rate
```

Comment:
Add the missing forward slash for the comment here `/frame rate` -> `//frame rate` . This causes a compilation failure in CI.